Classes/ExpressionLanguage/DocumentTypeFunctionProvider.php

```diff
@@ -54,10 +54,10 @@ public function getFunctions(): array
     /**
      * This holds the current document
      *
-     * @var Document|null
+     * @var Document
      * @access protected
      */
-    protected ?Document $document;
```

Comment:
This line is a cause for https://github.com/kitodo/kitodo-presentation/issues/1293